oem/ibm/libpldmresponder/file_io_type_pcie.cpp

```diff
@@ -600,6 +600,21 @@ void PCIeInfoHandler::parseSecondaryLink(
         std::filesystem::path mexConnecter(mexConnecterPath);
         auto mexSlotandAdapter = getMexSlotandAdapter(mexConnecter);
 
+        std::cout << "Creating associations under "
```

Comment:
do we need this explicitily?

Reply:
Yes, to make sure that associations gets created. This will be called only once after receiving Topology data.